src/Administration/Resources/app/administration/src/module/sw-customer/snippet/en-GB.json

```
@@ -31,7 +31,11 @@
     },
     "card": {
       "buttonImitateCustomer": "Log in as customer",
-      "tooltipImitateCustomerGuest": "Guest accounts cannot be imitated.",
+      "tooltipImitateCustomerGuest": "Guest customers cannot be imitated.",
+      "tooltipImitateCustomerInactive": "Inactive customers cannot be imitated.",
+      "tooltipImitateCustomerInactiveSalesChannel": "The customers Sales Channel is not active.",
+      "tooltipImitateCustomerNoDomain": "The Sales Channel of the customer has no domain configured.",
+      "tooltipImitateCustomerNoStorefront": "The Sales Channel of the customer is not of type Storefront.",
```

Might be nitpicking, so up to you, but I prefer if items like this are separated:

```suggestion
      "tooltipImitateCustomer": {
          "guest": "Guest customers cannot be imitated.",
          "inactive": "Inactive customers cannot be imitated.",
          "inactiveSalesChannel": "The customers Sales Channel is not active.",
          "noDomain": "The Sales Channel of the customer has no domain configured.",
          "noStorefront": "The Sales Channel of the customer is not of type Storefront."
      }
```